frontend/src/metabase/admin/settings/components/widgets/EmbeddingCustomizationInfo.tsx

```diff
@@ -6,9 +6,9 @@ import SettingHeader from "../SettingHeader";
 
 export const EmbeddingCustomizationInfo = () => {
   const setting = {
-    description: jt`In order to remove the Metabase logo from embeds, you can always upgrade to one of our ${(
+    description: jt`In order to remove the Metabase logo from embeds, you can always upgrade to ${(
       <ExternalLink href={MetabaseSettings.upgradeUrl()}>
```

Comment:
@WiNloSt the way `jt` works is that is requires using `key` for all jsx items, e.g. `<ExternalLink key="link"`. The key should be unique among other items in the enclosing `jt` expression. It also works without the `key` but it prints warnings to the console.

Reply:
Dang! I never knew. I'll fix those.

Reply:
Fixed.